### DISCUSSION TITLE
How to handle extension updates? Technically and in respect to new addons

### DISCUSSION THREAD

**Discussion by WorldLanguages** [ScratchAddons/ScratchAddons#65] [Discussion]
- Will new addons with `"enabled_by_default": true` be automatically enabled? Will the user be notified about this? If so, how?
- Have a release blog? Might solve the above issue.
- Might never happen, but if an addon wants to suddenly change how its options are structured, it will need help from the addon loader in order to restore previous settings, and not just throw them into the trash and use the default ones.
- How often will updates happen? Does a tiny bug fix in a rarely used addon make the addon deserve a global update?
- After the first release, should we have 2 branches? Dev and stable (stable would be the one available in stores, and we could do hotfixes on it).

**Comment by BoomerScratch** [comment]
> * After the first release, should we have 2 branches? Dev and stable (stable would be the one available in stores, and we could do hotfixes on it).

I think master and dev is better.

**Comment by WorldLanguages** [comment]
Lol, it was somehow a joke because "master" is not a very appropriate word apparently (even tho in Spanish, I'd say it's almost a compliment). Whatever, language is weird.
